cloud/amazon/iam_cert.py

```diff
@@ -105,9 +104,9 @@
 try:
     import boto
     import boto.iam
-    HAS_BOTO = True
 except ImportError:
-    HAS_BOTO = False
+    print "failed=True msg='boto required for this module'"
+    sys.exit(1)
```

Comment:
You should probably use something like this here instead:
```
fail_json(changed=False, msg='Python boto is required for this module')
```
http://docs.ansible.com/ansible/developing_modules.html#module-checklist